dts/common/yaml/i2c.yaml

```diff
@@ -0,0 +1,25 @@
+---
+title: I2C Base Structure
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives the base structures for all I2C devices
+
+properties:
+  - clock-frequency :
```

Comment:
Indeed, aim is to converge to Linux Kernel DT.
For node properties, it's better to stick with Linux naming, as long as this is not in complete contradiction with driver behavior in Zephyr.